components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/test/resources/swagger.json

```diff
@@ -0,0 +1,3135 @@
+{
```

Comment:
Does it become mandatory to duplicate this file? :( The content seems to be a bit old as well.

The problem is if we keep duplicates we need to keep all of them updated all the time. If someone changes the original file in a way that it breaks test cases, the tests will still pass if he does not update the swagger resource in the test classes. The next person to write a test case will immediately get the error as soon as he updated the swagger in test resources.